agent/lm_agent/parsing/rlm.py

```diff
@@ -1,70 +1,118 @@
 """
-Parser for rlm
+Parser for RLM
 """
 import re
+from typing import Optional
 
 INT = r"\d+"
 VERSION = rf"v{INT}\.{INT}"
 HOSTWORD = r"[a-zA-Z0-9-]+"
 HOSTWORD2 = r"[a-zA-Z0-9-.]+"
 HOSTNAME = rf"{HOSTWORD}(\.{HOSTWORD2})*"
+FEATURE_NAME = r"[a-zA-Z0-9-_]+"
```

Comment:
I think you can just use `[\w_]` for this.

Reply:
`[\w-]` worked, the `-` needs to be parsed since some RLM licenses have it in the feature name.

Reply:
Oh, sheesh....I used the wrong character in my suggestion. Good catch.